test/StartupPerf/startupPerf.ps1

```diff
@@ -0,0 +1,47 @@
+# execute this first: Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process -Confirm 
+
+function GetUrlStatusCode($url)
+{
+	$req = [system.Net.WebRequest]::Create($url)
+	try {
+		$res = $req.GetResponse()
+	} catch [System.Net.WebException] {
+		$res = $_.Exception.Response
+	}
+	return [int]$res.StatusCode
+}
+
+$totalTime = 0
+$attempts = 10
+
+function StartUnderDebugger()
+{
+	$script:sw = [system.diagnostics.stopwatch]::StartNew()
+
+	$dte.ExecuteCommand("Debug.Start");
```

Comment:
Debug.Start is mostly async operation. It is like a click on menu item. After this compilation and iisepress start happens. So I need to ensure that app is actually loaded. Typically it is just couple http requests I need to do to get result.

Stop debugger is not that interesting to measure. It may be good idea though to sleep a second before the next debug session start.